types/result.model.ts

```diff
@@ -0,0 +1,39 @@
+export interface Result {
+	id: string;
+	round_id: string;
+	team_id: string;
+	location_id: string;
+	deviation: number; // Is dit seconden?
+	pulses: null; // Is dit niet hetzelfde als 'times'?
+	status: number;
+	location: Location;
+	time: string; // Waarom worden hier ineens UNIX gegeven?
```

Comment:
We hebben twee type pulsen:

start pulse -> unix start tijd van de ploeg
anders pulsen zijn delta tijden